t/03-testapi.t

```diff
@@ -467,6 +467,19 @@ subtest 'assert_and_click' => sub {
     is_deeply($cmds->[-1], {cmd    => 'backend_mouse_set', x      => 100,     y   => 100}, 'assert_and_click succeeds and move to old mouse set');
 };
 
+subtest 'assert_and_tclick' => sub {
```

Comment:
This seems to be a near-duplicate of the `assert_and_dclick` functionality. To have better maintainable code I suggest to reduce some duplication. One thing you could easily do is to not duplicate the complete "subtest" block but instead extend the existing "assert_and_dclick" code to not need to redefine the mock again. E.g. like the following

```perl
subtest 'assert_and_dclick and assert_and_tclick' => sub {
    my $mock_testapi = Test::MockModule->new('testapi');
    $mock_testapi->redefine(assert_screen => {area => [{x => 1, y => 2, w => 3, h => 4}]});
    # assert_and_dclick tests …
    # assert_and_tclick tests …